packages/maestro/src/plugin/twig.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * This is a transferred version of `larowlan/vite-plugin-twig-drupal
```

Comment:
We have [forked](https://github.com/international-labour-organization/twig-processor) it but maintaining separate repos will be an additional pain. Since it's small enough, we decided to include it as a part of `maestro`

Reply:
@justintemps I thought it would be useful to have a copy of the original source somewhere since we are going to modify this copy of it and because we can't trust that the original repo would stay up.

But I guess this plugin is our code now so the original source is of little help.